admin/class-virgil-pure-admin.php

```diff
@@ -266,6 +267,10 @@ public function virgil_pure_profile_update(int $userId): void
         $wpUser = get_user_by('id', $userId);
         $this->updatePassword($wpUser);
     }
+    public function virgil_pure_user_register(int $userId): void
+    {
+        $this->fh->migrate();
```

Comment:
Add method: `migrateOneUser($userId)`, like this one - https://github.com/VirgilSecurity/virgil-pure-wordpress/blob/a0d852a528b0660836e5c1a028b4d86b2409fc1c/admin/core/Core/FormHandler.php#L161

Reply:
@SergeySeroshtan added and changed to new method